WordPress/Classes/ViewRelated/Blog/DomainCredit/DomainCreditRedemptionSuccessViewController.swift

```diff
@@ -0,0 +1,69 @@
+import UIKit
+
+@objc protocol DomainCreditRedemptionSuccessViewControllerDelegate {
+    func continueButtonPressed()
+}
+
+/// Displays messaging after user successfully redeems domain credit.
+class DomainCreditRedemptionSuccessViewController: UIViewController {
+    private let domain: String
+
+    private weak var delegate: DomainCreditRedemptionSuccessViewControllerDelegate?
+
+    init(domain: String, delegate: DomainCreditRedemptionSuccessViewControllerDelegate) {
+        self.domain = domain
+        self.delegate = delegate
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let attributedSubtitleConfiguration: NoResultsViewController.AttributedSubtitleConfiguration = {
+            [weak self] attributedText in
+            guard let domain = self?.domain else {
+                return nil
+            }
+            return self?.applyDomainStyle(to: attributedText, domain: domain)
+        }
+        let controller = NoResultsViewController.controllerWith(title: NSLocalizedString("Congratulations", comment: "Title on domain credit redemption success screen"),
+                                                                buttonTitle: NSLocalizedString("Continue", comment: "Action title to dismiss domain credit redemption success screen"),
+                                                                subtitle: nil,
+                                                                attributedSubtitle: generateDomainDetailsAttributedString(domain: domain),
+                                                                attributedSubtitleConfiguration: attributedSubtitleConfiguration,
+                                                                image: "wp-illustration-domain-credit-success",
+                                                                subtitleImage: nil,
+                                                                accessoryView: nil)
```

Comment:
These `nil` parameters when calling `NoResultsViewController.controllerWith` aren't necessary since they all have default values of `nil`.

Reply:
yup not needed! just removed